docs/reference/commandline/trust_sign.md

```diff
@@ -16,10 +16,14 @@ keywords: "sign, notary, trust"
 # trust sign
 
 ```markdown
-Usage:  docker trust sign IMAGE:TAG
+Usage:  docker trust sign [OPTIONS] IMAGE:TAG
```

Comment:
I think @riyazdf pulled `[OPTIONS]` out of a bunch of the other commands. Is it OK here?

Reply:
Yes, we pulled it out because they didn't have options yet. This one now does.